src/os/bluestore/NVMEDevice.cc

```diff
@@ -38,6 +38,7 @@
 #include "common/errno.h"
 #include "common/debug.h"
 #include "common/perf_counters.h"
+#include <boost/intrusive/slist.hpp>
```

Comment:
could you move this `#include` up before `#include <spdk/nvme.h>`. see https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes.

Reply:
Let update it.